changelog.d/4640.bugfix

```diff
@@ -0,0 +1,5 @@
+# [[Issue #4640]](https://github.com/vector-im/element-android/issues/4640) The notification badge jumps around
```

Comment:
for context, we're using https://github.com/twisted/towncrier to generate the release notes https://github.com/vector-im/element-android/releases and it automatically does the hyperlinking of issue in the file name for us :tada: